src/model/export-report/interfaces.ts

```diff
@@ -0,0 +1,16 @@
+export interface FilterStateInterface {
+  backupDestinationGuids: string[];
+  backupPolicyGuids: string[];
+  groupBy: string;
+  hypervisorClusterGuids: string[];
+  hypervisorGuids: string[];
+  hypervisorManagerGuids: string[];
+  projectGuids: string[];
+  virtualMachineGuids: string[];
+}
+
+export interface ExportRequestInterface {
```

Comment:
you can put these interfaces into different files, then export them via `index.ts`